### PR TITLE
[CN-1127] Improve setup of security group and firewall rules for AKS, EKS, GKE

### PR DESCRIPTION
**Pull request by cheels**
## Description
In the e2e test (including AWS, EKS ,Azure, Istio), we are allowing all IP addresses (0.0.0.0/0) as the source of network security. So this PR fixed this.

### CODE REVIEW

**Review by hasancelik — CHANGES_REQUESTED** [review]

File: .github/workflows/e2e-aks.yaml [hunk 1]
```diff
@@ -16,7 +16,13 @@ jobs:
     runs-on: ubuntu-latest
     outputs:
       CLUSTER_NAME: ${{ steps.create-cluster.outputs.CLUSTER_NAME }}
+      NRG: ${{ steps.create-cluster.outputs.NRG }}
+      NSG: ${{ steps.create-cluster.outputs.NSG }}
     steps:
+      - name: Get GH Public IP
```
[the remark this answers is not in the document]
Comment (cheels) [reply .github/workflows/e2e-aks.yaml]: Removed

**Review by semihbkgr — CHANGES_REQUESTED** [review]
We are using the same VPC for each test run in GKE platform.
So, It would be better to delete the `firewall-rules` after the execution in GKE and Istio workflows.

**Review by semihbkgr — APPROVED** [review]
LGTM 👍.
I have one question. Do we really need to set firewall rules for resilience tests?

**Comment by cheels** [comment]
> LGTM 👍. I have one question. Do we really need to set firewall rules for resilience tests?
You are right. Removed this part.

**Pull request merged**